onnxruntime/python/tools/transformers/microbench/benchmark.py

```diff
@@ -0,0 +1,30 @@
+import time
+import onnxruntime as ort
+
+
+def create_session(onnx_file, provider, profiling=False):
+    sess_opt = ort.SessionOptions()
```

Comment:
any case we want to config with more options?

Reply:
Probably ok for now; will add as needed.